app/src/main/res/values/strings.xml

```diff
@@ -48,24 +48,24 @@
   <string name="signup">Sign up</string>
   <string name="logging_in_title">Logging in</string>
   <string name="logging_in_message">Please wait…</string>
-  <string name="login_success">Login success!</string>
-  <string name="login_failed">Login failed!</string>
+  <string name="login_success">Logged in!</string>
+  <string name="login_failed">Cannot Login!</string>
   <string name="upload_failed">File not found. Please try another file.</string>
-  <string name="authentication_failed">Authentication failed, please login again</string>
+  <string name="authentication_failed">Error in authentication, please login again</string>
   <string name="uploading_started">Upload started!</string>
   <string name="uploading_queued">Upload queued (limited connection mode enabled)</string>
   <string name="upload_completed_notification_title">%1$s uploaded!</string>
   <string name="upload_completed_notification_text">Tap to view your upload</string>
   <string name="upload_progress_notification_title_start">Starting %1$s upload</string>
   <string name="upload_progress_notification_title_in_progress">%1$s uploading</string>
   <string name="upload_progress_notification_title_finishing">Finishing uploading %1$s</string>
-  <string name="upload_failed_notification_title">Uploading %1$s failed</string>
+  <string name="upload_failed_notification_title">Cannot upload %1$s</string>
   <string name="upload_paused_notification_title">Uploading %1$s paused</string>
   <string name="upload_failed_notification_subtitle">Tap to view</string>
   <string name="upload_paused_notification_subtitle">Tap to view</string>
   <string name="title_activity_contributions">My Recent Uploads</string>
   <string name="contribution_state_queued">Queued</string>
-  <string name="contribution_state_failed">Failed</string>
+  <string name="contribution_state_failed">Error detected</string>
```

Comment:
I think failed fits better for this specific usage. Error detected does not indicate that upload process failed, ie. we may have an error for adding categories (so error detected) and upload may be done. So, "error detected" is not necessarily mean "upload failed".

Suggestions are welcome, I am not the most native Ensglish speaker I know.